src/App.js

```diff
@@ -1,21 +1,71 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Provider } from 'react-redux';
+import { Route, Redirect, NavLink, Switch } from 'react-router-dom';
+
+import GifCard from './components/GifCard/GifCard';
+import Body from './containers/Body/Body';
+import SearchGif from './components/SearchGif/SearchGif';
+import ErrorBoundary from './components/ErrorBoundary/ErrorBoundary';
+import store from './store';
+import global from './style/global';
+import { Border, Title, Header } from './style/style';
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+	render() {
+		return (
+			<Provider store={store}>
+				<div>
+					<Header>
+						<Title>Madness</Title>
+						<ul>
+							<li>
+								<NavLink to="/Home">Home</NavLink>
```

Comment:
cool that you used NavLink, but you missed the activeClassName to actually pass the class to the Link when the route matches